src/main/kotlin/datamaintain/Config.kt

```diff
@@ -3,4 +3,5 @@ package datamaintain
 import java.nio.file.Path
 
 data class Config(val path: Path,
-                  val mongoUri: String)
\ No newline at end of file
+                  val mongoUri: String,
+                  val dbDriver: DatamaintainDriver? = null)
```

Comment:
We need dbdriver in constructor to test pruner. The parameter is optional to avoid immediately implementing the construction of this driver, which I believe will be the subject of another ticket.

Another solution could have been to make a stub (ex empty mongo driver) .

Reply:
I would prefer to have an empty driver.